src/main/kotlin/leets/leenk/global/auth/domain/service/AppleAuthService.kt

```
@@ -0,0 +1,258 @@
+        }
+    }
+
+    /**
+     * JSON 문자열을 Map으로 파싱
+     */
+    @Suppress("UNCHECKED_CAST")
+    private fun parseJson(json: String): Map<String, Any> =
+        try {
+            val objectMapper = ObjectMapper()
+            objectMapper.readValue(json, Map::class.java) as Map<String, Any>
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)